src/component/FeatureRenderer.js

```diff
@@ -184,6 +184,25 @@ Ext.define('GeoExt.component.FeatureRenderer', {
         symbolType: 'Polygon'
     },
 
+    statics: {
+
+        /**
+         * Determines the style for the given feature record.
+         *
+         * @param {GeoExt.data.model.Feature} record A feature record to get the
+         *     styler for.
+         * @return {ol.style.Style[]|ol.style.Style} The style(s) applied to the
+         *     given feature record.
+         * @private
+         */
+        determineStyle: function(record) {
+            var feature = record.olObject;
+            return feature.getStyle() || feature.getStyleFunction() ||
+                (record.store ? record.store.layer.getStyle() : null);
```

Comment:
It is copied from the old class, so should be fine for the moment. But I can dig into that afterwards, if you're OK with that.